desktop/src/app/components/navbar/warnings/modals/delete-outliers-modal.component.ts

```diff
@@ -39,6 +42,16 @@ export class DeleteOutliersModalComponent {
         if (event.key === 'Escape') this.activeModal.dismiss('cancel');
     }
 
+    public async initialize() {
+
+        this.foundDocuments = await this.datastore.find({
```

Comment:
I don't think it's necessary to fetch these documents during initialization as they are only used in "prepareDeleteAll".

Reply:
I changed this now in `delete-outliers-modal.component.ts` as well as `fix-outliers-modal.component.ts`